### ISSUE TITLE
Terraform version 0.15 panics on tf.Init

### ISSUE THREAD

**Issue by frouioui** [hashicorp/terraform-exec#163]
## Description

Hello👋 Terraform recently upgraded to 0.15, and ever since my program started failing when calling `tf.Exec()` with the following error:

```
-lock, -lock-timeout, -verify-plugins, and -get-plugins options are no longer available as of Terraform 0.15: unexpected version 0.15.0 (min: -, max: 0.15.0)
```

## Reproduce

The issue can be reproduced with the sample Go code:

```go
package main

import (
	"context"
	"fmt"
	"io/ioutil"
	"os"

	"github.com/hashicorp/terraform-exec/tfexec"
	"github.com/hashicorp/terraform-exec/tfinstall"
)

func main() {
	tmpDir, err := ioutil.TempDir("", "tfinstall")
	if err != nil {
		panic(err)
	}
	defer os.RemoveAll(tmpDir)

	execPath, err := tfinstall.Find(context.Background(), tfinstall.LatestVersion(tmpDir, false))
	if err != nil {
		panic(err)
	}

	workingDir := "/Users/pflorent/Project/arewefastyet_fork/infra/terraform"
	tf, err := tfexec.NewTerraform(workingDir, execPath)
	if err != nil {
		panic(err)
	}

	err = tf.Init(context.Background(), tfexec.Upgrade(true), tfexec.LockTimeout("60s"))
	if err != nil {
		panic(err)
	}

	state, err := tf.Show(context.Background())
	if err != nil {
		panic(err)
	}

	fmt.Println(state.FormatVersion) // "0.1"
}
```

The temporary fix I have found is to initialize `tf` using `tfinstall.ExactVersion()` with a version < 0.15, instead of `tfinstall.LatestVersion()`.

## Specs

Running on MacOS with go1.16.

**Comment by kmoe** [comment]
Thanks for letting us know. I've updated the example in the README now so it works by default with Terraform 0.15.

The panic is due to the `panic(err)` in the example code, not terraform-exec. The error message gives more information about why it failed:

```
-lock, -lock-timeout, -verify-plugins, and -get-plugins options are no longer available as of Terraform 0.15: unexpected version 0.15.0 (min: -, max: 0.15.0)
```

This means that `tfexec.LockTimeout("60s")` is not compatible with Terraform 0.15.

**Comment by frouioui** [comment]
Awesome, thank you.
